sebs/faas/function.py

```
@@ -134,28 +218,63 @@ def __init__(self):
-        )
+        # Ensure 'begin' and 'end' are present and are valid numbers before conversion
+        if "begin" in self.output and "end" in self.output:
+            try:
+                begin_ts = float(self.output["begin"])
+                end_ts = float(self.output["end"])
+                self.times.benchmark = int(
+                    (datetime.fromtimestamp(end_ts) - datetime.fromtimestamp(begin_ts))
+                    / timedelta(microseconds=1)
+                )
+            except (ValueError, TypeError) as e:
+                self.logging.error(f"Could not parse benchmark begin/end times from output: {e}")
```

Pylint (3.3.7)

[error] Instance of 'ExecutionResult' has no 'logging' member

(E1101)